docs/resources/metastore.md

```diff
@@ -52,6 +52,8 @@ The following arguments are required:
 * `name` - Name of metastore.
 * `storage_root` - Path on cloud storage account, where managed [databricks_table](table.md) are stored. Change forces creation of a new resource.
 * `owner` - (Optional) Username/groupname of Metastore owner.
+* `delta_sharing_enabled` - (Optional) Required along with `delta_sharing_recipient_token_lifetime_in_seconds`. Used to enable delta sharing on the metastore.
```

Comment:
@nkvuong from a resource end user perspective this field is set during the create portion of the lifecycle. Is this comment still relevant? As if the user sets this during creation it will still apply the patch after it is created. (in the code logic)

Reply:
This is trying to explain that  our API ignores those fields during creation so users have to explicitly re-run terraform apply again for it to take effect

Reply:
Ahh the create is written in a way that they don't have to rerun terraform apply. We call update during create in the createfunc. It's clever enough to call patch if user configured delta sharing during the create thanks to @nfx

Reply:
👀